edx/edx-enterprise
# ENT-608: Do not create baskets and orders for audit enrollments.

**Description:** 
This PR updates enterprise landing page logic to make sure we do not create basket with orders as part of audit enrollment.

**JIRA:** [ENT-608](https://openedx.atlassian.net/browse/ENT-608)

**Testing instructions:**

1. Create a course with multiple modes including an audit mode.
2. Make sure course is associated with an enterprise customer.
3. Create an enterprise learner and open enterprise landing page for above course and enterprise for the above created learner
4. Learner should be enrolled in the course with audit mode.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
